hail/src/main/scala/is/hail/backend/local/LowerTableIR.scala

```diff
@@ -56,7 +56,7 @@ object LowerTableIR {
       ir.copy(ir.children.map(lower))
   }
 
-  def lower(tir: TableIR): LocalTableIR = tir match {
+  def lower(tir: TableIR): LocalTableIR = (tir: @unchecked) match {
```

Comment:
Don't we actually want to have a rule for ever table ir here? Is this a work-in-progress thing and when we're done we'll remove the unchecked?

Reply:
yes, although I'd use
```
case _ => throw new cxx.UnsupportedCXXException("unimplemented node: $node") 
```
or something here, as in the SparkBackend lowerer. (Echoing Dan, I'd generally prefer something even just like `case _ => <relevant error>` when it's not immediately obvious why certain cases are omitted.)